lldb/test/Shell/SymbolFile/DWARF/x86/discontinuous-inline-function.s

```diff
@@ -42,6 +27,21 @@ look_me_up:
         .size   foo.__part.1, .Lfoo.__part.1_end-foo.__part.1
 
 
+bar:
+        nop
+.Lbar_end:
+        .size   bar, .Lbar_end-bar
+
+        .type   foo,@function
+foo:
```

Comment:
Trying to understand the change here.

I think what you have done is moved them around so that the order is:
foo.__part.1
bar
foo

"look_me_up" is inside of foo.__part.1, which puts it below foo. Previously this would have lead to the error, but not it does not.

Either because look_me_up's range would be below foo's low pc, or same thing for foo.__part.1, not sure which.

Reply:
(and yes, `look_me_up` is in the middle of that block, so that it causes lldb to parse it)

Reply:
Ok I understand now. `look_me_up` is used to trigger something, rather than having special properties itself.